sdk/openai/Azure.AI.OpenAI/api/Azure.AI.OpenAI.netstandard2.0.cs

```diff
@@ -620,7 +623,9 @@ public partial class ChatMessageImageContentItem : Azure.AI.OpenAI.ChatMessageCo
     }
     public partial class ChatMessageImageUrl : System.ClientModel.Primitives.IJsonModel<Azure.AI.OpenAI.ChatMessageImageUrl>, System.ClientModel.Primitives.IPersistableModel<Azure.AI.OpenAI.ChatMessageImageUrl>
     {
+        public ChatMessageImageUrl(string dataUri) { }
```

Comment:
imageUri

Reply:
I thin imageUri is confusing and ambiguous, since it could be understood as a Uri to an image, but I called it dataUri to make clear that a [DataUri](https://en.wikipedia.org/wiki/Data_URI_scheme?wprov=sfla1) is expected here.